movement/watch_faces/complication/totp_face_lfs.c

```diff
@@ -40,6 +40,7 @@ struct totp_record {
     size_t secret_size;
     char label[2];
     uint32_t period;
+    hmac_alg algorithm;
```

Comment:
Probably nice to explicitly initialise the algorithm to SHA1 in `init_totp_record` below.

Reply:
You are right :). This was fixed in [ca7806a](https://github.com/joeycastillo/Sensor-Watch/pull/151/commits/ca7806a79b806e1a82373e9c805f2d5b80a33f7c).